.gitignore

```diff
@@ -18,3 +18,89 @@ Recipe.conf
 *.deb
 *.tar.gz
 *.tar.xz
+
+### VisualStudioCode template
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+*.code-workspace
+
+# Local History for Visual Studio Code
+.history/
+
+### Linux template
+*~
+
+# temporary files which can be created if a process still has a handle open of a deleted file
+.fuse_hidden*
+
+# KDE directory preferences
+.directory
+
+# Linux trash folder which might appear on any partition or disk
+.Trash-*
+
+# .nfs files are created when an open file is removed but is still being accessed
+.nfs*
+
+### Ninja template
+.ninja_deps
+.ninja_log
```

Comment:
I don't know the purpose of most of these .gitignore entries, but I think I can voice an opinion on this one.

These two files will only get created inside the build directory you configured. You should really ignore that entire directory, including the build.ninja file itself and a bunch of stuff in meson-private/ etc... but you don't have to, because Meson sets this up for you automatically: https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

So you don't need to explicitly ignore these two files either.